<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit_window</class>
 <widget class="QMainWindow" name="Credit_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="line_summ"/>
          </item>
          <item>
           <widget class="QLabel" name="label_summ">
            <property name="text">
             <string>Сумма кредита</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="line_day"/>
          </item>
          <item>
           <widget class="QLabel" name="label_day">
            <property name="text">
             <string>Срок кредита (в месяцах)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="line_proc"/>
          </item>
          <item>
           <widget class="QLabel" name="label_proc">
            <property name="text">
             <string>Процентная ставка</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="calc_Button">
            <property name="text">
             <string>Расчитать</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radio_dif_type">
              <property name="text">
               <string>Дифференцированный</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_anue_type">
              <property name="text">
               <string>Аннуитетный</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_month_pay">
            <property name="text">
             <string>Ежемесячный платеж</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_overpay">
            <property name="text">
             <string>Переплата по кредиту</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_all_payed">
            <property name="text">
             <string>Общая выплата</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_res_month_pay">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_res_overpay">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_res_all_payed">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Тип</string>
    </property>
    <addaction name="main_calc_button"/>
    <addaction name="Credit_calc_button"/>
    <addaction name="depos_calc_button"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="main_calc_button">
   <property name="text">
    <string>Математический</string>
   </property>
  </action>
  <action name="Credit_calc_button">
   <property name="text">
    <string>Кредитный</string>
   </property>
  </action>
  <action name="depos_calc_button">
   <property name="text">
    <string>Депозитный</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
